lib/rules/lint-no-invalid-interactive.js

```diff
@@ -111,8 +111,8 @@ module.exports = class InvalidInteractive extends Rule {
           return;
         }
 
-        // Allow onerror={{action "foo"}} on img tags
-        if (this._element.tag === 'img' && node.name === 'onerror') {
+        // Allow onerror={{action "foo"}} and onload={{action "foo"}} on any tag
+        if (node.name === 'onerror' || node.name === 'onload') {
```

Comment:
What do you think about this:

```suggestion
        if (this._element.tag === 'img' && ['onerror', 'onload'].includes(node.name)) {
```

Specifically, we only want to allow `onload` and `onerror` properties on `<img>`'s (AFAIK) and using `[].includes()` makes the logic a tiny bit easier to read.

Reply:
I've applied the change and squashed.